packages/ui/src/ui/components/ACL/ACL-connect-helpers.js

```diff
@@ -67,6 +68,7 @@ const makeAclMapStateToProps = (inputIdmKind) => {
             disableInheritanceResponsible,
         } = state.acl[idmKind];
 
+        const hasApprovers = getHasApprovers(state, idmKind);
         const approvers = getApproversOrderedByInheritanceAndSubject(state, idmKind);
```

Comment:
`approvers` contains result of filtering, it is better to reflect this in the variable name. `getApproversFiltered`/`getApproversFilteredAndOrdered` would be more descriptive, without this, it is not clear why we need to call `getHasApprovers` here.